introduction.md

```diff
@@ -0,0 +1 @@
+"hi! although this is not me first time using git. but i am using it just to try and learn once again" 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.